content/applications/finance/fiscal_localizations/italy.rst

```diff
@@ -49,52 +51,145 @@ section, click :guilabel:`Update info`. From here, fill out the fields:
+keep track of the exact :guilabel:`Tax Exemption Kind (Natura)` and :guilabel:`Law Reference` that
+justify the exemption operated on an invoice line.
 
-You can check the current status of an invoice by the :guilabel:`Electronic invoicing` field. The
-XML file can be found in the **chatter** of the invoice.
+For example the tax for export in the EU to be used as reference (`0% EU`, invoice label `00eu`),
+which can be found under :menuselection:`Accounting --> Configuration --> Taxes`. Exports are exempt
+from VAT, and therefore they require the :guilabel:`Has exoneration of tax (Italy)` option ticked,
+with both the :guilabel:`Exoneration` kind and :guilabel:`Law Reference` filled in.
 
-.. image:: italy/italy-test.png
+.. figure:: italy/italy-tax.png
    :align: center
-   :alt: Electronic invoicing status (waiting for confirmation)
+   :alt: Tax Exemption Settings
+
+   Tax Exemption Settings
 
 .. seealso::
-   :doc:`../accounting/customer_invoices/electronic_invoicing`
+   There are many :guilabel:`Tax Exemption Kind (Natura)` and :guilabel:`Law Reference` codes.
+   Make sure you are selecting the correct one by reading the official documentation (in Italian)
+   here.
+
+   - `Documentation portal with latest link <https://www.agenziaentrate.gov.it/portale/web/guest/aree-tematiche/fatturazione-elettronica>`_
+   - `Official guide on Tax exemption codes v1.8 <https://www.agenziaentrate.gov.it/portale/documents/20143/451259/Guida_compilazione-FE-Esterometro-V1.8.pdf/c0aca469-4c4a-9b8d-048c-398412435c26>`_
+
+   .. figure:: italy/italy-tax-agency-docs.png
+     :align: center
+     :alt: Tax Exemption codes documentation
+
+     Tax Exemption codes documentation
+
+.. note::
+   If you need to use a different kind of :guilabel:`Exoneration`, click :menuselection:`Action -->
+   Duplicate` within the tax menu to create a copy of an existing similar tax. Then, select another
+   :guilabel:`Exoneration`, write the related :guilabel:`Law Reference` and :guilabel:`Save`.
+   Repeat this process as many times as you need different kind of :guilabel:`Exoneration` taxes.
 
-File processing authorization (Odoo)
-------------------------------------
+.. tip::
+   **Rename** your taxes in the :guilabel:`Name` field according to their :guilabel:`Exoneration` to
+   differentiate them easily.
+
+.. _italy/reverse-charge:
+
+Reverse charge
+==============
+
+Italian businesses selling goods and services are sometimes required *not* to charge the customer
+for the VAT. The customers pay the VAT *themselves* to the :abbr:`AdE (Agenzia delle Entrate)`
+instead. This mechanism is called **reverse charge**.
+
+- :guilabel:`Internal Reverse Charge` (for domestic sales) was born to fight Tax evasion if the VAT
+  is paid by the seller, especially in sectors like waste management, construction, subcontracting
+  in the construction sector, the sale of precious metals, as well as transactions involving mobile
+  phones and phone cards.
```

Comment:
Mhm, I was thinking of customers going to the page and not knowing in advance - I thought the list of sectors would help. I'm following your advice, in case we can have feedback about that.